### DISCUSSION TITLE
👀➕ How about a visit count on your profile?

### DISCUSSION THREAD

**Discussion by GenilsonDC** [community/community#126492] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### I couldn't be sure about the visits to my profile.

I participated in selection processes that asked for my GitHub Profile.
**I couldn't be sure about the visits to my profile.**
I was frustrated and anxious.
The times I went to my page I didn't see any 👁‍🗨 **Unwatch** 2 or more.
 

> I believe that for those who are starting out or contributing to the community, they want to check profile visits.

 **It makes sense?**

**Comment by GenilsonDC** [comment]
Desde já obrigado e desculpa os erro de **english grammar**
💜

**Comment by davevad93** [comment, marked as answer]
Hi @GenilsonDC ,

Check this:
[github-profile-views-counter](https://github.com/antonkomarev/github-profile-views-counter)  

You can add a badge to your profile `README` that shows the visitors count, for example:

| Demo| Code|
| ----- | ---- |
| ![](https://komarev.com/ghpvc/?username=GenilsonDC&style=for-the-badge) | `![](https://komarev.com/ghpvc/?username=GenilsonDC&style=for-the-badge)` |

**Reply by GenilsonDC** [reply]
👑 THANNKSSSZ 👑

<a href="https://github.com/antonkomarev/github-profile-views-counter">
    <img src="https://komarev.com/ghpvc/?username=GenilsonDC&color=00224D&style=for-the-badge">
</a>

**Reply by davevad93** [reply]
You're welcome!
